packages/terra-core-docs/src/terra-dev-site/test/tag/TagListTags.test.jsx

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-alert */
+import React from 'react';
+import IconBookmark from 'terra-icon/lib/icon/IconTile';
+import IconTag from 'terra-icon/lib/icon/IconTag';
+import Tag from 'terra-tag';
+import TagList from 'terra-tag/src/TagList';
```

Comment:
We should _NOT_ use an internal path to import in a file (I know some Terra components use this pattern but it is a bad one).

We should create an index.js file that exports the subcomponent and then we can import it as:

```js
import { TagList } from 'terra-tag';
```